server/play/count_crib.go

```diff
@@ -44,7 +44,7 @@ func (*cribCountingHandler) HandleAction(
 
 	if cca.Pts != pts {
 		addPlayerToBlocker(g, pID, model.CountCrib, pAPIs, `you did not submit the correct number of points for the crib`)
-		return nil
+		return errors.New(`wrong number of points`)
```

Comment:
without returning an error here we end up persisted games with bad state. :feelsbatman: